test/differential_equations.jl

```diff
@@ -1,5 +1,8 @@
 @testset "Successful run with DE.jl" begin
-    f = (u,p,t) -> (p*u)
-    prob_ode_linear = ODEProblem(f,Float32sr(1.0)/Float32sr(2.0),(Float32sr(0.0),Float32sr(1.0)),Float32sr(1.01));
-    sol =solve(prob_ode_linear,Tsit5())  
+    for SR in [Float32sr, Float16sr, BFloat16sr]
+        f = (u,p,t) -> (p*u)
+        prob_ode_linear = ODEProblem(f,SR(1.0)/SR(2.0),(SR(0.0),SR(1.0)),SR(1.01));
+        sol =solve(prob_ode_linear,Tsit5())  
+        @test eltype(sol) == SR
+    end
```

Comment:
While this is obviously good to test, I'm not sure this should be part of our test suite. If DE.jl changes then we may get failed tests and ideally the two packages should be independent of each other (despite obv working together). So I'm temped to set these tests to skip, to disable them by default going forward. What do you think?

Reply:
Yeah I think that makes sense. It's a test we can call if needed